tools/test.py

```diff
@@ -121,6 +121,7 @@ def main():
     is_device_cpu = (args.device == 'cpu')
     device_id = None if is_device_cpu else parse_device_id(args.device)
 
+    destroy_model = model.destroy
```

Comment:
not all `End2EndModel` has `destroy` method.

Reply:
A `destroy` method is also added in `BaseBackendModel`

https://github.com/open-mmlab/mmdeploy/blob/1d2ca0d232a2a9e52b9eb59934e99b83f2fe8254/mmdeploy/codebase/base/backend_model.py#L122-L124